com.playeveryware.eos/Runtime/Core/PlatformManager.cs

```diff
@@ -22,6 +22,7 @@
 
 namespace PlayEveryWare.EpicOnlineServices
 {
+    using JetBrains.Annotations;
```

Comment:
Ooooo mystery. What is this for?

Reply:
Nothing, added accidentally. Removed now.